tox.ini

```
@@ -14,9 +14,9 @@ deps =
     pytest-pylint
     pytest-pythonpath
     pytest-runner
-    django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<1.10
-    django110: Django>=1.10,<1.11
+    django32: Django>=3.2,<3.3
+    django42: Django>=4.2,<4.3
```

```suggestion
    django32: Django~=3.2.0
    django42: Django~=4.2.0
```
this will pick the latest patch release of the specified version range